pkg/occlient/occlient.go

```diff
@@ -83,8 +84,8 @@ type Client struct {
 	serviceCatalogClient servicecatalogclienset.ServicecatalogV1beta1Interface
 	routeClient          routeclientset.RouteV1Interface
 	userClient           userclientset.UserV1Interface
-	kubeConfig           clientcmd.ClientConfig
-	namespace            string
+	KubeConfig           clientcmd.ClientConfig
```

Comment:
Why is this uppercase? It looks like a case of find / replace. You should revert this back to normal. The lowercasing doesn't match with the other variable names.

Reply:
Yeah it was required as part of addressing https://github.com/redhat-developer/odo/pull/740#discussion_r225461742 and a similar situation even with mocking config related tests due to SetCurrentProject